docs/convolution/index.rst

```diff
@@ -153,7 +153,14 @@ algorithm, while :func:`~astropy.convolution.convolve_fft` uses a Fast Fourier
 Transform (FFT). Thus, the former is better for small kernels, while the latter
 is much more efficient for larger kernels.
 
-For example, to convolve a 1D dataset with a user-specified kernel, you can do::
+Example
+-------
+
+..
+  EXAMPLE START
+  Convolution for User-Specified Kernels
```

Comment:
@jonathansick - just out of curiosity, is there any way we can make the markup more concise, e.g.

```
.. EXAMPLE START: Title

...

.. EXAMPLE END
```

?

Reply:
That'd be fine. The comment syntax is just a human-readable/searchable placeholder for the actual syntax, which is

```rst
.. example:: title

   Example content.
```

Reply:
Good to know! So I should change the comment markup to:

.. EXAMPLE START: Title

Content

.. EXAMPLE END

with just one colon after "example start" correct?

Reply:
Sounds good :) Starting by next week at the latest I'll start pushing again to get the extension shipped so you can use the real syntax at some point.